shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveyUI.kt

```diff
@@ -0,0 +1,11 @@
+package co.nimblehq.blisskmmic.presentation.model
+
+data class SurveyUI (
```

Comment:
Android naming suggestion
```suggestion
data class SurveyUiModel (
```

Reply:
Nice catch, rebase conflict. Removed.